docs/tutorials/dronecan_flight_controller.rst

```diff
@@ -40,13 +50,12 @@ the module are still accurate for DroneCAN.
 
 Node ID
 ---------
-The node ID is a number the motor will use to identify itself on the DroneCAN bus. **The node ID must be unique for each module, and none of the 
+The node ID is a number the module will use to identify itself on the DroneCAN bus. **The node ID must be unique for each module, and none of the 
 modules should use the same node ID as the flight controller.** Typically PX4 flight controllers will use node ID 1 by default, and ArduPilot flight controllers will 
-use node ID 10 by default, so it is best to avoid those IDs. ID 0 is reserved for other DroneCAN features. You should 
-assign the node ID of each module to some unique value from 1 to 127 excluding your flight controller's node ID. By default, the node ID of a Vertiq module is set to 99.
+use node ID 10 by default, so it is best to avoid those IDs.
 
-**Note that your flight controller's documentation may mention dynamic node ID allocation with DroneCAN. Vertiq modules do not currently support dynamic node ID allocation, you must configure a 
-static node ID for each module through the Control Center.** 
+If you are connecting your module a previously configured bus with unknown node IDs, or simply want to avoid setting specific node IDs for each of your modules, you can use :ref:`DroneCAN's Dynamic Node ID Allocation feature supported by Vertiq modules <dynamic_node_id_allocation>`. 
```

Comment:
If you are connecting your module _to_ a previously configured bus with unknown node IDs....